src/index.js

```diff
@@ -0,0 +1,12 @@
+// import { fetchCharacterData, getCharacterImagesAndIds } from './modules/fetchApi.js';
```

Comment:
- Optional
Kindly remove the uncommented code to keep the code base clean. Generally, it is a mandatory requirement, but I am marking it as optional since you are meeting all other requirements. I believe you will follow-up on this. 👍